.registryrc

```diff
@@ -1 +1 @@
-7c6bfefbaefb6aad19b114afb6743b1b70c69552
+179d801b80ec36b6dd54cb8adcf25e1c9e50db97
```

Comment:
@tkporter - we should set this to the commit after we merge the corresponding registry PR?

Reply:
yep!